README.md

```diff
@@ -1,407 +1,436 @@
-
-<C-c>        close telescope
-<Esc>        close telescope (in normal mode)
-```
-
-To see the full list of mappings, check out `lua/telescope/mappings.lua` and the `default_mappings` table.
-
-To override ALL of the default mappings, you can use the `default_mappings` key in the `setup` table.
+As an example of using the `setup()` method,
+  the following code configures `telescope.nvim`
+  to its default settings.
 
+```lua
+require('telescope').setup{
+  defaults = {
+    vimgrep_arguments = {
+      'rg', 
+      '--color=never', 
+      '--no-heading', 
+      '--with-filename', 
+      '--line-number', 
+      '--column', 
+      '--smart-case'
+    },
+    prompt_position = "bottom",
+    prompt_prefix = ">",
+    selection_strategy = "reset",
+    sorting_strategy = "descending",
+    layout_strategy = "horizontal",
+    layout_defualts = {
+      -- TODO add builtin options.
+    },
+    file_sorter =  require'telescope.sorters'.get_fuzzy_file ,
+    file_ignore_patterns = {},
+    generic_sorter =  require'telescope.sorters'.get_generic_fuzzy_sorter,
+    shorten_path = true,
+    winblend = 0,
+    width = 0.75,
+    preview_cutoff = 120,
+    results_height = 1,
+    results_width = 0.8,
+    border = {},
+    borderchars = { '─', '│', '─', '│', '╭', '╮', '╯', '╰'},
+    color_devicons = true,
+    use_less = true,
+  }
+}
 ```
- To disable a keymap, put [map] = false
 
-        So, to not map "<C-n>", just put 
+To embed the above code snippet in a `.vim` file
+  (for example in `after/plugin/telescope.nvim.vim`),
+  wrap it in `lua << EOF code-snippet EOF`:
 
-            ...,
-            ["<C-n>"] = false,
-            ...,
-
-        Into your config.
+```lua
+lua << EOF
+require('telescope').setup{
+  -- ...
+}
+EOF
+````
 
- Otherwise, just set the mapping to the function that you want it to be.
+#### Mappings
 
-            ...,
-            ["<C-i>"] = actions.goto_file_selection_split
-            ...,
+Mappings are fully customizable.
+Many familiar mapping patterns are setup as defaults.
 
+| Action                           | Mappings       |
+|----------------------------------|----------------|
+| Next item                        | `<C-n>/<Down>` |
+| Previous item                    | `<C-p>/<Up>`   |
+| Next/previous (in normal mode)   | `j/k`          |
+| Confirm selection                | `<CR>`         |
+| go to file selection as a split  | `<C-x>`        |
+| go to file selection as a vsplit | `<C-v>`        |
+| go to a file in a new tab        | `<C-t>`        |
+| scroll up in preview window      | `<C-u>`        |
+| scroll down in preview window    | `<C-d>`        |
+| close telescope                  | `<C-c>`        |
+| close telescope (in normal mode) | `<Esc>`        |
 
-```
+To see the full list of mappings, check out `lua/telescope/mappings.lua` and
+the `default_mappings` table.  
 
-A full example:
+To change default mapping globally, then change default->mappings dict
```

Comment:
?

Reply:
You are right doesn't make any sense to I've taken it out. the below code snippet does better job at explaining how to change the mappings.